### DISCUSSION TITLE
Is it possible to create a new PR every time I run backport command ?

### DISCUSSION THREAD

**Discussion by Alspirid** [Mergifyio/mergify#5070] [Q&A]
I got a PR as a result of backport command, but I had to close it and deleted that branch.
When I'm trying to run backport command again on the source PR, mergify points me to the same PR which I closed.

**Comment by rfguimaraes** [comment]
I had the exactly same issue today and I could not find a way to create a new PR for backports with mergify :disappointed:

**Reply by jd** [reply]
What's wrong with reopening the PR though?

**Reply by fruch** [reply]
that it won't start the process from scratch

example, I have a conflict in a backport, cause some other change is missing.
I backport that missing change in a different backport PR

now I want to start the original PR from scratch, ontop of the updated branch.

none of the commands I've tried helped, `@mergify rebase` just make thing worse, and you get all of the conflict duplicated.

I had to pull each PR (we have multiple branch we backport into), and do the process manually
